### DISCUSSION TITLE
Is the spooky kids looking at the camera still in the .fla file?

### DISCUSSION THREAD

**Discussion by ghost** [KadeArchive/Kade-Engine#564] [Q&A] [answered]
I'm trying to make it so that this frame:
![image](https://user-images.githubusercontent.com/61301511/119370236-0d037480-bc83-11eb-9132-95c5a81f63b1.png)
plays when the spooky kids say yeah or haha, kinda like Tankman's ugh animation, and I swear that it used to be in the .fla file, but I can't find it anymore.

**Comment by ghost** [comment]
Check Funkin source and look in old versions

**Reply by ghost** [reply]
not there

**Reply by ghost** [reply]
Maybe it never was then

**Reply by ghost** [reply]
Hold up...
You have the frame, you can add it

**Reply by SarahDrewington** [reply]
The frame in question is a screenshot from PA's twitter, you'd have to recreate it as its not in the FLA's or base files, sorry

**Reply by ghost** [reply]
could you not add the above frame to the fla?

**Reply by SarahDrewington** [reply]
No, as it is a PNG, and it would display with its gray bg and be pixelated.

**Reply by ghost** [reply]
It may be pixelated but the grey BG could be removed

**Reply by SarahDrewington** [reply]
Not in flash, it'd have to be removed in an image editing program, then imported, not to mention you'd also need the inbetweens like BF's peace anim.

**Comment by SarahDrewington** [comment]
Try and recreate the frame and its possibly inbetweens inside of Flash, then code it in.

**Comment by pepperly-art** [comment]
Would tossing [this ](https://www.newgrounds.com/dump/item/34cbc922ee82afc6074ee89f4abeeada)in the spookykids file work for you?

**Reply by SarahDrewington** [reply]
Whoa, where is this from?

**Reply by pepperly-art** [reply]
There's a reason my UN ends in -art lmao

**Reply by SarahDrewington** [reply]
well damn, im blind but this is also super well made, like i genuinely though this was official

**Reply by pepperly-art** [reply]
No worries and thanks! It probably helps that I started from the above png, haha. I hope it'll fool more people in the future 🤣

**Comment by Burger-Gaming** [comment, marked as answer]
I think i have the FLA for the spookykids file where the animation is there actually

**Reply by Burger-Gaming** [reply]
nevermind I guess it was never in the FLA file, sorry for getting your hopes up

**Reply by Burger-Gaming** [reply]
Might as well contribute since i didnt, I made a spritesheet and XML for @pepperly-art 's fla animation. ill link a google drive [here.](https://drive.google.com/file/d/1RS4apay-p_NInJKBDLaAYmKb9cZ0JAOZ/view?usp=sharing)

**Reply by Burger-Gaming** [reply]
It's gonna need to be coded in like BF's hey animation, but I think it would work

**Reply by ghost** [reply]
I know the coding part, so thanks for the spritesheet
